mappings/net/minecraft/util/math/intprovider/UniformIntProvider.mapping

```diff
@@ -11,7 +11,9 @@ CLASS net/minecraft/class_6019 net/minecraft/util/math/intprovider/UniformIntPro
 		ARG 0 instance
 	METHOD method_35017 create (II)Lnet/minecraft/class_6019;
 		ARG 0 min
+			COMMENT the minimum value, inclusive
 		ARG 1 max
+			COMMENT the maximum value, inclusive
```

Comment:
Hmm, this'd already be clear if the class used the codec names (`min_inclusive` and `max_inclusive`) instead of just `min` and `max`. (Same for `BiasedToBottomIntProvider`.)